spec/controllers/articles_controller_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 RSpec.describe ArticlesController, type: :controller do
-  describe '#index' do
+  describe 'GET #index' do
     describe 'show the list to articles saved' do
     end
   end
 
-  describe '#show' do
+  describe 'GET #show' do
     it 'show the articles' do
     end
   end
 
-  describe '#new'do
+  describe 'GET #new'do
     it 'create new post' do
```

Comment:
`it 'assigns an article' do`

Methods `GET` don't persist to database, they only assign variables.